src/chrome/content/rules/AIA-Surety.xml

```diff
@@ -1,18 +1,15 @@
 <ruleset name="AIA Surety">
 
 	<target host="aiasurety.com" />
-	<target host="*.aiasurety.com" />
+	<target host="becomeanagent.aiasurety.com" />
+	<target host="www.aiasurety.com" />
 	<target host="expertbail.com" />
 	<target host="www.expertbail.com" />
 
 
-	<securecookie host="^(?:.*\.)?(?:aiasurety|expertbail)\.com$" name=".+" />
+	<securecookie host=".+" name=".+"/>
```

Comment:
```suggestion
	<securecookie host=".+" name=".+" />
```

`sed -e 's/<securecookie host=".+" name=".+"\/>/<securecookie host=".+" name=".+" \/>/g'`

Reply:
a better approach is to modify the script to include the whitespace. however, I want to avoid re-doing the same PR and add complications to the review process. Will submit another PR to fix this later. thanks.